src/main/java/dev/ebullient/convert/tools/dnd5e/Tools5eJsonSourceCopier.java

```diff
@@ -196,48 +196,57 @@ public JsonNode mergeSubrace(JsonNode subraceNode, JsonNode raceNode) {
         return subraceOut;
     }
 
-    // 	utils.js: static getCopy (impl, copyFrom, copyTo, templateData,...) {
     @Override
-    protected JsonNode mergeNodes(Tools5eIndexType type, String originKey, JsonNode copyFrom, JsonNode copyTo) {
-        // edit in place: if you don't, lower-level copies will keep being revisted.
-        ObjectNode target = (ObjectNode) copyTo;
-
-        JsonNode _copy = MetaFields._copy.getFrom(copyTo);
-        JsonNode _mod = _copy == null ? null : normalizeMods(MetaFields._mod.getFrom(_copy));
-        JsonNode _trait = _copy == null ? null : MetaFields._trait.getFrom(_copy);
-
+    protected JsonNode handleTrait(JsonNode copyInfo) {
+        JsonNode traitNode = MetaFields._trait.getFrom(copyInfo);
+        // Not actually used in any current sources, but is defined in the schema.
+        if (traitNode == null) {
+            return null;
+        }
         JsonNode templateApplyRoot = null;
+        // fetch and apply external template mods
+        String templateKey = Tools5eIndexType.monsterTemplate.createKey(traitNode);
+        JsonNode template = getOriginNode(templateKey);
+        if (template == null) {
+            tui().warn("Unable to find trait for " + templateKey);
+        } else {
+            template = copyNode(template); // copy fast
 
-        if (_trait != null) {
```

Comment:
This changed to _template, and I missed that. I'll have to update and test for those values to make sure they are still applied